Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Composable/NolPay/NolPayGetList.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  NolPayGetLinkedCardsComponent.swift
+//  PrimerSDK
+//
+//  Created by Boris on 15.9.23..
+//
+
+import Foundation
+import PrimerNolPaySDK
+
+public class NolPayGetLinkedCardsComponent: PrimerHeadlessComponent {
+    
+    private var nolPay: PrimerNolPay!
+    public var errorDelegate: PrimerHeadlessErrorableDelegate?
+    
+    public init() {
+        guard let nolPaymentMethodOption = PrimerAPIConfiguration.current?.paymentMethods?.first(where: { $0.internalPaymentMethodType == .nolPay})?.options as? MerchantOptions,
+              let appId = nolPaymentMethodOption.appId
+        else {
+            let error = PrimerError.generic(message: "Initialisation error",
+                                            userInfo: [
+                                                "file": #file,
+                                                "class": "\(Self.self)",
+                                                "function": #function,
+                                                "line": "\(#line)"
+                                            ],
+                                            diagnosticsId: UUID().uuidString)
+            ErrorHandler.handle(error: error)
+            self.errorDelegate?.didReceiveError(error: error)
+            return
+        }
+        
+        guard let clientToken = PrimerAPIConfigurationModule.decodedJWTToken else {
+            let err = PrimerError.invalidClientToken(userInfo: ["file": #file, 
+                                                                "class": "\(Self.self)",
+                                                                "function": #function,
+                                                                "line": "\(#line)"],
+                                                     diagnosticsId: UUID().uuidString)
+            ErrorHandler.handle(error: err)
+            return
+        }
+        
+        nolPay = PrimerNolPay(appId: appId, isDebug: true, isSandbox: true) { sdkId, deviceId in
```

Comment:
Would you like to control `isDebug` and `isSandbox` with some conditions like if we are in release variant then developers don't want to leak log and if Primer's environment is `PRODUCTION` then `isSandbox` should be `false`